frontend-migration/README.md

```
@@ -0,0 +1,756 @@
+```
+
+**해결:** 파일 상단에 `'use client'` 추가
+```typescript
+'use client'
+
+import { useRouter } from 'next/navigation'
+```
+
+### 2. QueryClient 에러
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)